app/javascript/components/Articles/ArticlesHeader.tsx

```diff
@@ -1,21 +1,21 @@
-import React from 'react';
+import React from "react";
 
 export interface ArticlesHeaderProps {
-  title:string,
-  subtitle:string,
-  link:string
+  title: string;
+  subtitle: string;
+  link: string;
 }
- 
+
 const ArticlesHeader: React.FC<ArticlesHeaderProps> = (props) => {
-  return ( 
+  return (
     <div id="articles-header">
-      <a href={props.link} >
+      <a href={props.link} data-testid="article-header-link">
```

Comment:
Similar to last comment. I think you could query this with `'div#articles-header a'`